wiki/Oppia-Android-Code-Coverage.md

```diff
@@ -0,0 +1,375 @@
+
+The data presented includes:
+
+1. **File:** Name of the file that failed to meet the coverage threshold (Clicking the drop-down reveals the file's path in the codebase).
+2. **Coverage Percentage:** Percentage of coverage achieved for the file.
+3. **Lines Covered:** Number of lines covered by the test cases, shown in "lines hit / lines found" format.
+4. **Status:** Indicates whether the coverage status is a pass or fail.
+5. **Required Percentage:** Minimum percentage required to pass the coverage check for the file.
+
+#
+
+**2.c. Passing coverages**
+
+### Passing coverage
+
+<details open>
+<summary>Files with passing code coverage</summary><br>
+
+| File | Coverage | Lines Hit | Status | Min Required |
+|------|:--------:|----------:|:------:|:------------:|
+| <details><summary>Pass.kt</summary>utility/src/main/java/org/oppia/android/util/Pass.kt</details> | 94.26% | 197 / 209 | :white_check_mark: | 70% |
+</details>
+
+#
+
+This section highlights files that have met the minimum coverage requirements. Files with overridden coverage thresholds are marked with an asterisk (*), and their specific required percentages are provided.
+
+These files have meet their overridden minimum coverage to achieve a passing status.
+
+For a complete list of files with overridden coverage thresholds, refer to the [test_file_exemptions.textproto](https://github.com/oppia/oppia-android/blob/develop/scripts/assets/test_file_exemptions.textproto) file.
+
+The data presented includes:
+
+1. **File:** Name of the file that met the coverage threshold (Clicking the drop-down reveals the file's path in the codebase).
+2. **Coverage Percentage:** Percentage of coverage achieved for the file.
+3. **Lines Covered:** Number of lines covered by the test cases, shown in "lines hit / lines found" format.
+4. **Status:** Indicates whether the coverage status is a pass or fail.
+5. **Required Percentage:** Minimum percentage required to pass the coverage check for the file.
+
+**2.d. Exempted coverages**
+
+### Exempted coverage
+<details open><summary>Files exempted from coverage</summary> <br>
+
+| File | Failure Reason |
+|------|----------------|
+| <details><summary>TestExemptedFile.kt</summary>app/src/main/java/org/oppia/android/app/activity/TestExemptedFile.kt</details> | This file is exempted from having a test file; skipping coverage check. |
+| <details><summary>SourceIncompatible.kt</summary>app/src/main/java/org/oppia/android/app/activity/SourceIncompatible.kt</details> | This file is incompatible with code coverage tooling; skipping coverage check. |
+
+</details>
+
+#
+
+Certain files are exempt from coverage checks. These exemptions include:
+
+1. **Test File Exemptions:** Files that do not have corresponding test files are exempt from coverage checks. Since no test files are available for these sources, coverage analysis cannot be performed, and these files are therefore skipped.
```

Comment:
```suggestion
1. **Test File Exemptions:** Files are exempted from having corresponding test files are also exempt from coverage checks. Since no test files are available for these sources, coverage analysis cannot be performed, and these files are therefore skipped.
```

Clarification since there's another interpretation for the previous wording: that tests are simply absent (which is actually a failure when not exempted, as you discuss in the page above).

Reply:
Yeah, updated to be more specific to exempted cases. Thanks